.github/workflows/build.yml

```diff
@@ -91,3 +91,16 @@ jobs:
       maven-args: package -Pkubernetes -DskipUTs -DskipITs -DskipBuildImage=${{ needs.changes.outputs.rss_server_docker == 'false' }}
       cache-key: package
       go-version: '1.17'
+
+  kubernetes-integration:
+    needs: [kubernetes]
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Compile kubernetes operator
+      run: |
+        mvn package -Pkubernetes -DskipTests
+    - name:  Run local Integration Tests
```

Comment:
These steps are also needed.
```
    - name: Checkout project
      uses: actions/checkout@v3
    - name: Set up JDK ${{ inputs.java-version }}
      uses: actions/setup-java@v3
      with:
        java-version: ${{ inputs.java-version }}
        distribution: ${{ inputs.jdk-distro }}
    - name: Cache local Maven repository
      if: ${{ inputs.cache-key != '' }}
      uses: actions/cache@v3
      with:
        path: ~/.m2/repository
        key: mvn-${{ inputs.java-version }}-${{ inputs.cache-key }}-${{ hashFiles('**/pom.xml') }}
        restore-keys: |
          mvn-${{ inputs.java-version }}-${{ inputs.cache-key }}-
    - name: Set up Go ${{ inputs.go-version }}
      if: ${{ inputs.go-version != '' }}
      uses: actions/setup-go@v3
      with:
        go-version: ${{ inputs.go-version }}
```
If kubernetes module code isn't changed, the kubernetes ci won't be triggered. So you can add a empty line in the file `https://github.com/apache/incubator-uniffle/blob/master/deploy/kubernetes/operator/api/uniffle/v1alpha1/remoteshuffleservice_types.go` to trigger kubernetes CI.

Reply:
My fault. If we add empty line, k8s ci will fail because of code style. So I add a blank in the comment. Now, we can run the k8s ci.

Reply:
There still are some other errors. Could you fix them?